### DISCUSSION TITLE
Suddenly Unable to Log In / Total App Meltdown

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#12505] [Help] [answered]
Well, my boss reached out to me again saying Budibase is inoperable. 

We're running Budibase Cloud. No changes made in weeks to our app. Functional earlier today during work hours. Now I'm getting the following weird sequence of failures to login with Google SSO, then a successful login, then when accessing the app, a total meltdown and logged back out. 

https://github.com/Budibase/budibase/assets/99847728/9c61f34c-351e-4bd1-9ae6-10283e9ab02d

Here is a screenshot of the error popups right before being kicked out of the session. 

![Screenshot 2023-12-05 180934](https://github.com/Budibase/budibase/assets/99847728/71c677a2-56cb-43cd-bf31-45b1272fda71)

How do we fix this and am I the only one who has consistent and absolute App meltdown every couple of months? Am I the only one experiencing this issue? Last time it was a user type change that caused the problem, I see there's a new Creator type user in the November release... could that be the issue?

I need some help here as I'm now being asked to explore alternatives to Budibase which is so frustrating. I love the Budibase interface, the intuitiveness and the community but paying for a product that is mission critical and yet fails so often has become very difficult to substantiate to my boss.

**Comment by stellarfinn** [comment]
Here's a second screenshot from another video in which I was able to successfully login for a short period of time and received a different error.

![image](https://github.com/Budibase/budibase/assets/99847728/caae2471-7566-4325-ac84-360ef98f7d78)

@melohagan @joebudi 

I also cannot login with a username and password as it says its invalid. I also cannot request a password reset. Every time I request a password reset, Budibase does not send any email with a password reset.  I assume because of the Google SSO.

**Reply by ConorWebb96** [reply]
Hey @stellarfinn,

I've responded to you on discord with some questions, as well as what the next steps we need from you are. Heres the [link](https://discord.com/channels/733030666647765003/1176565832525025382/1181861315233468457) to the thread in the off chance you can't find it!

Thanks in advance for the additional information.

**Comment by melohagan** [comment, marked as answer]
Hi @stellarfinn 

Apologies for the temporary downtime. An engineer has been investigating the issue today, and some minor tweaks have been made to our cloud environment that should hopefully see some improvement going forward.
